engine/lua/src/cs/dmsdk/lua/lauxlib_gen.cs

```diff
@@ -0,0 +1,184 @@
+// Generated, do not edit!
+// Generated with cwd=/Users/mathiaswesterdahl/work/defold/engine/lua and cmd=../../scripts/dmsdk/gen_sdk.py -i ./sdk_gen.json
+
+// Copyright 2020-2024 The Defold Foundation
+// Copyright 2014-2020 King
+// Copyright 2009-2014 Ragnar Svensson, Christian Murray
+// Licensed under the Defold License version 1.0 (the "License"); you may not use
+// this file except in compliance with the License.
+// 
+// You may obtain a copy of the License, together with FAQs at
+// https://www.defold.com/license
+// 
+// Unless required by applicable law or agreed to in writing, software distributed
+// under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+// CONDITIONS OF ANY KIND, either express or implied. See the License for the
+// specific language governing permissions and limitations under the License.
+
+using System.Runtime.InteropServices;
+using System.Reflection.Emit;
+
+using static dmSDK.Lua.Lua;
+
+namespace dmSDK.Lua {
+    public unsafe partial class LuaL
```

Comment:
I'm happy that 95% of Lua was possible to generate via our scripts.
And we get a `Lua` and `LuaL` class to separate the functions.